delfin/drivers/utils/ssh_client.py

```diff
@@ -218,10 +222,15 @@ def get(self):
                     return conn
                 else:
                     conn.close()
-            return self.create()
+                    self.current_size -= 1
         if self.current_size < self.max_size:
-            created = self.create()
-            self.current_size += 1
+            try:
+                self.current_size += 1
```

Comment:
Why +1 at first then -1 when exception happens? This would made that in a short time, the size is not correct(already +1 but not created, just creating).
Why not keep the old way that +1 after self.create()?

Reply:
done